test/behavior/tendrl/login.feature

```diff
@@ -0,0 +1,45 @@
+Feature: LOGIN
+
+  Scenario: Type username then delete
+    Given I open the Tendrl url
```

Comment:
Please right scenario in perspective of a user not "I"
example. user opens the Tendrl url

Reply:
`Given: a user is at tendrl login page`

instead of 

`I open the Tendrl url`